src/modules/subscription/reducer.ts

```diff
@@ -86,6 +89,13 @@ export function subscriptionReducer(state = buildInitialState(), action: Subscri
         error
       }
     }
+
+    case CLEAR_SUBSCRIPTIONS_ERROR: {
```

Comment:
Would you mind adding a test for the handling of the clear subscriptions error action?